packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -989,7 +989,9 @@ export class BasicCrawler<Context extends CrawlingContext = BasicCrawlingContext
         await this._init();
         await this.stats.startCapturing();
         const periodicLogger = this.getPeriodicLogger();
-        await this.setStatusMessage('Starting the crawler.', { level: 'INFO' });
+        // Don't await, we don't want to block the execution
+        this.setStatusMessage('Starting the crawler.', { level: 'INFO' })
+            .catch((err) => this.log.warning(`Error setting status message: ${err.message}`));
```

Comment:
this is not necessary, the `setStatusMessage` crawler method already handles failures, you can just skip the await

```suggestion
        // Don't await, we don't want to block the execution
        void this.setStatusMessage('Starting the crawler.', { level: 'INFO' });
```